jinahub/indexers/DocCache/README.md

```diff
@@ -1,14 +1,11 @@
 # DocCache
 
-**DocCache** is an Executor that can cache documents that it has seen before, by different combination of fields (or just one field). It then removes the Document that has the same combination of values in those fields from the DocumentArray, so it will not reach the following Executors in your Flow. 
-
-This is useful for continuously indexing Documents, and not having to worry about indexing the same Document twice.
+**DocCache** is an Executor that can cache documents that it has seen before by using the hashing based on one or more fields. It removes the Document that has the same values in those fields. So that the downstream Executors will not get duplicated Documents. This is useful for continuously indexing Documents when the same Document might appear for multiple times.
 
 ## Notes
 The Executor only removes Documents in the `/index` endpoint. In the other endpoints, operations are done by the Document `id`.
 
 
-
 ## Usage 
 
 In a Flow:
```

Comment:
Shouldn't we remove the usage from here?

Reply:
I kept the `Usage` part for this executor because it is not clear how to use it from the docstrings. Plus there is no related documentation at docs.jina.ai.